stable/gitlab-runner/values.yaml

```diff
@@ -0,0 +1,81 @@
+## GitLab Runner Image
+## ref: https://hub.docker.com/r/gitlab/gitlab-runner/tags/
+##
+image: gitlab/gitlab-runner:alpine-v1.10.4
+
+## Specify a imagePullPolicy
+## 'Always' if imageTag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## The GitLab Server URL (with protocol) that want to register the runner against
+## ref: https://docs.gitlab.com/runner/commands/README.html#gitlab-runner-register
+##
+# gitlabURL: http://gitlab.your-domain.com/
```

Comment:
You got a not matching value here: gitlabURL in default values but gitlabUrl in all templates.
Just discovered this while testing.